engage/app/templates/calendar.hbs

```diff
@@ -1,7 +1,7 @@
 <div class="row">
   <div class="col-md-8 main-left">
     <h2 class="page-header">Calendar</h2>
-    <div class="media list-group-item engage-list-group-item calendar-item">
```

Comment:
I thought we decided with the calendar items that it's best to keep these as `list-group-item engage-list-group-item` in order to have the nice padding + borders eh?